data/homepage.yml

```diff
@@ -76,129 +76,229 @@ tab:
       venue : "Room 5"
+      column: 6
       time : "11:00 AM - 11:25 AM"
+      start_hour: 11
+      start_minute: 0
+      end_hour: 11
+      end_minute: 25
 
     - speaker : "Shane Utt"
       speaker_image : "images/speakers/shane-utt-thumb.jpg"
       subject : "Kubernetes Networking reImagined"
       venue : "Room 4"
+      column: 5
       time : "11:35 AM - 12:00 PM"
+      start_hour: 11
+      start_minute: 35
+      end_hour: 12
+      end_minute: 0
 
     - speaker : "Wavell Watson"
       speaker_image : "images/speakers/wavell-watson-thumb.png"
       subject : "5g Core Testing with the CNF TestSuite"
       venue : "Room 5"
+      column: 6
       time : "11:35 AM - 12:00 PM"
+      start_hour: 11
+      start_minute: 35
+      end_hour: 12
+      end_minute: 0
 
     ## 12:00 PM
 
     - subject : "Lunch"
       time : "12:00 PM - 1:30 PM"
+      start_hour: 12
+      start_minute: 0
+      end_hour: 13
+      end_minute: 30
+      column: -1
 
     ## 1:00 PM
 
     - speaker : "Angel Ramirez"
       speaker_image : "images/speakers/angel-ramirez-thumb.png"
       subject : "Kubernetes Kickstart Boosting Agility with CNCF and DevOps"
       venue : "Room 5"
+      column: 6
       time : "1:00 PM - 2:30 PM"
+      start_hour: 13
+      start_minute: 0
+      end_hour: 14
+      end_minute: 30
 
     - speaker : "TBD"
       speaker_image : "images/speakers/speaker-thumb-one.jpg"
       subject : "RHEL Security (Sponsor: Red Hat)"
       venue : "Room 2"
+      column: 3
       time : "1:30 PM - 4:15 PM"
+      start_hour: 13
+      start_minute: 30
+      end_hour: 16
+      end_minute: 15
 
     - speaker : "TBD"
       speaker_image : "images/speakers/speaker-thumb-one.jpg"
       subject : "Smart Contract Driven Development (Sponsor: Rezoomex)"
       venue : "Room 3"
+      column: 4
       time : "1:30 PM - 4:15 PM"
+      start_hour: 13
+      start_minute: 30
+      end_hour: 16
+      end_minute: 15
 
     ## 2:00 PM
 
     - speaker : "Rafa Brito"
       speaker_image : "images/speakers/rafa-brito-thumb.jpg"
       subject : "Demystifying Kubernetes Resource Management"
       venue : "Room 5"
+      column: 6
       time : "2:40 PM - 4:10 PM"
+      start_hour: 14
+      start_minute: 40
+      end_hour: 16
+      end_minute: 10
 
     ## 4:00 PM
 
     - speaker : "Mark Lavi"
       speaker_image : "images/speakers/mark-lavi-thumb.jpg"
       subject : "Navig8.dev: a Helm Chart Values Wizard"
       venue : "Room 4"
+      column: 5
       time : "4:15 PM - 4:20 PM"
+      start_hour: 16
+      start_minute: 15
+      end_hour: 16
+      end_minute: 20
 
     - speaker : "Rajeshkumar Sampathrajan"
       speaker_image : "images/speakers/rajeshkumar-sampathrajan-thumb.jpg"
       subject : "Be Up for PCA success"
       venue : "Room 4"
+      column: 5
       time : "4:15 PM - 4:20 PM"
+      start_hour: 16
+      start_minute: 15
+      end_hour: 16
+      end_minute: 20
```

Comment:
These two overlap and are two short to render properly. Will there be talks this short?

Reply:
That's my mistake, the second one should be 4:20 PM to 4:25 PM, as per the [KCD schedule](https://community.cncf.io/events/details/cncf-kcd-texas-presents-kcd-texas-2024/).  As far as I know the duration is correct.  If rendering them is an issue, what about lumping them together as "KCD lightning talks" from 4:15 PM to 4:25 PM?

Reply:
Rendering them isn't an issue but having enough space to fit even a ten-minute slot in takes a lot of room for the whole schedule. Are we going to have abstracts to link to at some point? Do you think we should just have a huge schedule?

Reply:
I double checked the KCD schedule, and they have a TBD slot from 4:25 PM to 4:30 PM.  I checked out this branch locally and replaced the short sessions with one "KCD lightning talks" session from 4:15 PM to 4:30 PM.  It renders well enough to read the title, but you have to scroll for the time.

![Screenshot from 2024-03-13 11-19-26](https://github.com/TXLF/2024website/assets/12187228/540199a3-ff1f-41e7-897b-dd9f18549da8)

The abstracts are all in PaperCall.  I do think we should figure out a place to put them on the site, but I don't know where that would be.